constructorio-client/src/test/java/io/constructor/client/ConstructorIOBrowseTest.java

```diff
@@ -92,6 +127,37 @@ public void BrowseShouldReturnAResultWithBrandFacets() throws Exception {
         assertTrue("browse result id exists", response.getResultId() != null);
     }
 
+    @Test
+    public void SearchShouldReturnAResultWithVariations() throws Exception {
+        ConstructorIO constructor = new ConstructorIO("", "key_dKjn8oS8czBw7Ebv", true, null);
+        UserInfo userInfo = new UserInfo(3, "c62a-2a09-faie");
+        BrowseRequest request = new BrowseRequest("Color", "Blue");
+        BrowseResponse response = constructor.browse(request, userInfo);
+        assertEquals("browse results exist", response.getResponse().getResults().size(), 30);
+        assertEquals("browse result [variations] exists", response.getResponse().getResults().get(0).getVariations().size(), 8);
+        assertEquals("browse result variation [facets] exists", response.getResponse().getResults().get(0).getVariations().get(0).getData().getFacets().size(), 8);
+        assertEquals("browse result variation [value] exists", response.getResponse().getResults().get(0).getVariations().get(0).getValue(), "Coat Aspesi blue");
+        assertEquals("browse result variation [variation id] exists", response.getResponse().getResults().get(0).getVariations().get(0).getData().getVariationId(), "M0E20000000ECTT");
+        assertEquals("browse result variation [url] exists", response.getResponse().getResults().get(0).getVariations().get(0).getData().getUrl(), "https://demo.commercetools.com/en/aspesi-coat-I502997385098-blue.html");
+        assertEquals("browse results count as expected", (int) response.getResponse().getTotalNumberOfResults(), 562);
+        assertTrue("browse result id exists", response.getResultId() != null);
+    }
+
+    @Test
+    public void SearchShouldReturnAResultWithSortOptions() throws Exception {
```

Comment:
Should this be `SearchShouldReturnAResultWithSortOptions `?

Reply:
yup, thanks for catching that. updated